arm-network/2016-06-01/network.json

```diff
@@ -6515,14 +6601,143 @@
       ],
       "description": "NetworkSecurityGroup resource"
     },
+	"EffectiveNetworkSecurityGroup": {
```

Comment:
Looks like the spacing is off for most of this section.  In General Tabs should 2 spaces